### ISSUE TITLE
Docker method throws error

### ISSUE THREAD

**Issue by 4444dogs** [mayudev/revcord#4]
I followed the instructions on `README.md` for Docker exactly yet when I ran `docker-compose up -d` I just got the following error.

```
ERROR: The Compose file is invalid because:
Service revcord has neither an image nor a build context specified. At least one must be provided.
```

**Comment by mayudev** [comment]
Should be fixed in 23d8d8a991b2043f0a082cf0bd525ec3310aacaf
Although this issue never occurred for me in Docker Compose version v2.6.1. If it still doesn't work, try using this version.

**Comment by 4444dogs** [comment]
Seems to work now so I will close this issue
